### ISSUE TITLE
feat: sbb-card / sbb-group refactoring

### ISSUE THREAD

**Issue by jeripeierSBB** [lyne-design-system/lyne-components#1833]
With this refactoring the sbb-group and sbb-card should be merged into one component. To make the card interactive, a <sbb-card-action> element can be placed inside the card. This solves accessibility issues and enables also slotting other actions or interactive content into a card.

Moreover, the new dashed variant should be included.

To be synchronized with #1242 

Remaining Todos:
 - [x] Nested buttons in card examples / home stories should be static @jeripeierSBB 
 - [x] Naming of dashed property align with Figma
 - [x] Clean up card mixin
 - [x] Align timetable row with card refactoring and document it @dauriamarco 
 - [ ] Testing with screen readers and browser combination
 - [x] Check high contrast mode @dauriamarco 
 - [x] Documentation @jeripeierSBB 
 - [ ] Migration docs
 - [x] UX check (Manuel)
 - [x] Acesssibility Check (Mürsel)
